tests/encrypted_lvm.spec.ts

```diff
@@ -66,6 +73,6 @@ test.describe('The main page', () => {
                     if (error.constructor.name !== "TimeoutError") throw (error);
                 }
             }
-        });
+      });
     });
-});
+});
```

Comment:
if vscode keeps one extra line at the end by default let's keep it please.

Reply:
Added the extra line, vscode does not care with/without the last blank line.